tools/sof-dump-status.py

```diff
@@ -340,6 +341,31 @@ def _covertType(type_lst):
         print(card_info['longname'])
         exit(0)
 
+    # The kernel has changed the default firmware path when community
+    # key is used. Here we output firmware path according to kernel's
+    # match table.
+    if ret_args.get('fwpath') is True:
+
+        def community_path(dmi, board_name, board_vendor):
+            path = "/lib/firmware/intel/sof/community"
+            # the same match table as the kernel
+            vendor_list = ["AAEON", "Google"]
+            board_list = ["UP-APL01"]
+            if board_name in board_list or board_vendor in vendor_list:
+                return path
+            else:
+                return ""
+
+        fw_path = "/lib/firmware/intel/sof"
+        sysinfo.loadDMI()
+        board = sysinfo.dmi["board_name"]
+        vendor = sysinfo.dmi["board_vendor"]
+        cpath = community_path(sysinfo.dmi, board, vendor)
+        if cpath != "":
+            fw_path = cpath
```

Comment:
why need the function?

Reply:
Xiuli said we may have more folders in the future, eg, production,  with a function, the program structure will be clearer